src/main/scala/com/redis/sentinel/SentinelMonitor.scala

```diff
@@ -111,13 +118,17 @@ class SentinelMonitor (address: SentinelAddress, listener: SentinelListener, con
 
 trait SentinelHeartBeater extends Runnable with Log{
   private var running = false
+  private val beatingSentinel = new AtomicBoolean(true) // initialised to true so we don't unnecessarily wake up ops
```

Comment:
why atomic boolean? it looks this private variable is only accessed by single thread